pkg/controller/rest/verifiable/operation_test.go

```diff
@@ -625,6 +625,7 @@ func TestGeneratePresentation(t *testing.T) {
 	t.Run("test generate presentation - error", func(t *testing.T) {
 		var jsonStr = []byte(`{
 			"name" : "sample",
+			"signatureType":"Ed25519Signature2018",
```

Comment:
You needed a new field?

Reply:
The function was modified. To get the expected error we need to provide `signatureType `. https://github.com/hyperledger/aries-framework-go/pull/2037/files#diff-957d4786eb551ebc07beb3497e685537R846

Reply:
@rolsonquadras @sudeshrshetty ^^^^